locale/en.json

```diff
@@ -479,7 +479,7 @@
 		"checkout-block.Åland-Islands": "Åland Islands",
 		"subscriptions-block.credit-card-information": "Payment Method and Information",
 		"subscriptions-block.payment-error": "There was an error processing your payment.",
-		"subscriptions-block.payment-information": "Credit card information",
-		"subscriptions-block.submit-payment": "Purchase Subscription"
+		"subscriptions-block.submit-payment": "Purchase Subscription",
+		"subscriptions-block.payWithCard-label": "Or Pay with card"
```

Comment:
@edwardcho1231 ,  when adding a new translation key, we need it added for all languages. The guide for doing so is here: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/2538275032/Arc+Themes+Blocks+Internationalisation.

Reply:
We don't have access to Lokalise themes project so I cannot upload files into the project. We are not changing an existing string but adding a new one as it's requested by the design.